test/distributed/_tensor/test_redistribute.py

```diff
@@ -366,7 +366,7 @@ def test_redistribute_shard_dim_change(self):
                 local_out_dt = out_dt.to_local()
                 local_expected_dt = expected_dt.to_local()
                 self.assertEqual(out_dt.to_local(), expected_dt.to_local())
-                if self.device_type == "cuda":
+                if self.device_type == "cuda" or self.device_type == "hpu":
```

Comment:
I wonder if we could truly generalize this line than adding another device string here?